doc/sphinx/introduction.rst

```diff
@@ -60,16 +60,18 @@ Nonetheless, the manual has some structure that is explained below.
    of the formalism. Chapter :ref:`themodulesystem` describes the module
    system.
 
--  The second part describes the proof engine. It is divided in six
+-  The second part describes the proof engine. It is divided in seven
    chapters. Chapter :ref:`vernacularcommands` presents all commands (we
    call them *vernacular commands*) that are not directly related to
    interactive proving: requests to the environment, complete or partial
    evaluation, loading and compiling files. How to start and stop
    proofs, do multiple proofs in parallel is explained in
    Chapter :ref:`proofhandling`. In Chapter :ref:`tactics`, all commands that
    realize one or more steps of the proof are presented: we call them
-   *tactics*. The language to combine these tactics into complex proof
-   strategies is given in Chapter :ref:`ltac`. Examples of tactics
+   *tactics*. The legacy language to combine these tactics into complex proof
```

Comment:
```suggestion
   *tactics*. The original language to combine these tactics into complex proof
```

I think it's premature to call ltac "legacy" when ltac2 is still considered experimental.

Reply:
The point is to already start discouraging people from using it, at least those who are not afraid by the experimental status of Ltac2. Despite being experimental, Ltac2 should still provide a better and more robust experience to people wanting to develop complex proof strategies.